CHANGELOG.md

```diff
@@ -45,9 +46,6 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 
 ### Fixed
 - Changes in customized entry types are now directly reflected in the table when clicking "Apply" or "OK"
-- Fixed [#593](https://github.com/JabRef/jabref/issues/593): Reference list generation works for OpenOffice/LibreOffice again
```

Comment:
Because there are identical ones further down (sort of my bad that they showed up there, as I changed the format of them, but somehow some other PR managed to restore the old ones...).